deploy/role.yaml

```diff
@@ -42,9 +42,24 @@ rules:
 - apiGroups:
   - core.kubefed.k8s.io
   resources:
-  - KubeFedCluster
+  - kubefedclusters
   verbs:
   - "*"
+- apiGroups:
+  - core.kubefed.k8s.io
+  resources:
+  - kubefedclusters/status
+  verbs:
+  - update
+- apiGroups:
+  - user.openshift.io
+  resources:
+  - users
+  - identities
+  verbs:
+  - list
+  - watch
+  - create
```

Comment:
We also need get and update. But Nurali is adding it in https://github.com/codeready-toolchain/member-operator/pull/26/